Ingestion/templates/python.ecs.template.yaml

```diff
@@ -481,6 +538,57 @@ Resources:
               - Name: EXECUTOR_ID
                 Value: !Ref ExecutorId
               - !Ref "AWS::NoValue"
+            - !If 
+              - UseOptionalEnvVar01
+              - Name: !Select [0, !Split ["=", !Ref OptionalEnvVars]]
+                ValueFrom: !Select [1, !Split ["=", !Ref OptionalEnvVars]]
```

Comment:
Should be Value, not ValueFrom

ValueFrom is used to resolve values from external sources

Reply:
(Same for all below)

Reply:
Oh, snap! Thank you for catching this. I copy-pasted the portion pertaining to optional secrets to speed things up and totally forgot to change that! Will do so momentarily 🙇‍♀️

Reply:
Done